src/assets/themes.css

```diff
@@ -52,3 +52,11 @@ Pull Requests will require passing at least WCAG AA requirements to be accepted
   --link: #82e79a;
   --text: white;
 }
+
+.own-theme{
```

Comment:
Instead of using `own-theme`, maybe use something like `arduino`, `arduino-theme`, or `arduino-light`?

Reply:
sry

Reply:
![image](https://user-images.githubusercontent.com/29358791/65975076-b30dc580-e433-11e9-8871-7f952de2f3bd.png)